### ISSUE TITLE
[Rule RSRCH]: <span> elements marked up with the role "definition" and an "aria-labelledby" attribute raise a false positive violation

### ISSUE THREAD

**Issue by Mark-Falconbridge-i2** [IBMa/equal-access#2416]
### Rule description

[test.html](https://github.com/user-attachments/files/23212397/test.html)

Running the accessibility checker on the attached file raises:

`The ARIA attributes "aria-labelledby" are not valid for the element <span> with ARIA role "definition"` violations referencing `Rule ID: aria_attribute_valid` and `Reason ID: Fail_invalid_role_attr`.

Yet, according to https://www.w3.org/TR/wai-aria-1.2/#definition, it is recommended to use aria-labelledby to refer to the associated element with the role "term".

This approach is also suggested here - https://www.digitala11y.com/definition-role/ where I took and modified the example from.

(I'm aware there are other issues in the attached file, it was just a simple repro case to prove the point).

### Relevant requirement(s)

4.1.2 Name, Role, Value

**Comment by tombrunet** [comment]
ARIA has changed this. See ARIA 1.3 where name is prohibited on definition. https://www.w3.org/TR/wai-aria-1.3/#definition
